tests/proofs/dexter/dexter-spec.md

```diff
@@ -49,6 +49,46 @@ If any of the conditions are not satisfied, the call fails.
       orBool IsUpdating
 ```
 
+## Set Baker
+
+The contract sets its delegate to the value of `baker` (and optionally freezes the baker to that particular value) if the following conditions are satisfied:
+
+1.  the token pool is _not_ currently updating (i.e. `storage.selfIsUpdatingTokenPool = false`)
+2.  exactly 0 tez was transferred to this contract when it was invoked
+3.  the txn sender is the `storage.manager`
+4.  the baker is _not_ already frozen
+
+FIXME: This claim is incomplete because it depends on the `operations` cell which we do not have yet.
+
+```
+  claim <k> #runProof(_IsFA2, SetBaker(_Baker, FreezeBaker)) => . </k>
```

Comment:
LGTM, but could we just specify the stack change for now until we have the `operations` cell? To make sure the proof goes through.

Reply:
nevermind, that would get messy with the storage updates.